envi/archs/arm/emu.py

```diff
@@ -274,6 +274,7 @@ def executeOpcode(self, op):
         # NOTE: If an opcode method returns
         #       other than None, that is the new pc
         try:
+            self.setFlag(PSR_T_bit, (op.iflags & envi.ARCH_MASK) in (envi.ARCH_THUMB, envi.ARCH_THUMB16))
```

Comment:
It seems fast and cleaner to section off the `(envi.ARCH_THUMB, envi.ARCH_THUMB16)` into a ready made constant in envi/archs/arm/constants.py or something.

Reply:
so... it would have to live in envi.__init__. are you ok with that?  
i try to import as little as possible into any const.py, so it's most often the 'top level' of modules, depending on nothing.  i never want to worry about const files being a part of import loops.

Reply:
alternately, i'm happy to put it higher in this file.

Reply:
done